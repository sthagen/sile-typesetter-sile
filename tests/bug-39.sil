\begin[papersize=a5,class=book]{document}
\nofolios
\neverindent
\use[module=packages.frametricks]
\right-running-head{First Line of Header\skip[height=6pt] Second Line of Header}
\showframe[id=all]

First page text
\break \skip[height=22cm]
Second page text

\end{document}
