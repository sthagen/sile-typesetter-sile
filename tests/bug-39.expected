Set paper size 	419.5275636	595.275597
Begin page
Mx 	34.8208
My 	75.6731
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	41 76 85 86 87 w=18.7549	(First)
Mx 	55.8072
T	83 68 74 72 w=19.3848	(page)
Mx 	77.4235
T	87 72 91 87 w=16.5332	(text)
Mx 	34.8208
My 	28.9983
T	41 76 85 86 87 w=18.7549	(First)
Mx 	55.8048
T	47 76 81 72 w=17.6074	(Line)
Mx 	75.6414
T	82 73 w=8.1396	(of)
Mx 	86.0102
T	43 72 68 71 72 85 w=29.5264	(Header)
Mx 	34.8208
My 	46.9983
T	54 72 70 82 81 71 w=29.4971	(Second)
Mx 	66.5458
T	47 76 81 72 w=17.6074	(Line)
Mx 	86.3812
T	82 73 w=8.1396	(of)
Mx 	96.7488
T	43 72 68 71 72 85 w=29.5264	(Header)
New page
Mx 	58.7339
My 	76.6203
T	54 72 70 82 81 71 w=29.4971	(Second)
Mx 	90.4613
T	83 68 74 72 w=19.3848	(page)
Mx 	112.0764
T	87 72 91 87 w=16.5332	(text)
End page
Finish
